Sprint-2/implement/tally.js

```diff
@@ -1,3 +1,11 @@
-function tally() {}
+function tally(listOfItems) {
+  console.log(typeof listOfItems)
+  if (typeof listOfItems !== 'object') throw new Error('Not a valid input')
+  const itemsCount = {};
+  for (const item of listOfItems) {
+    itemsCount[item] ? (itemsCount[item] += 1) : (itemsCount[item] = 1);
```

Comment:
This works, for better readability, can you please try another way for value assignment?

please refer: https://www.sitepoint.com/javascript-ternary-operator/

Reply:
The link refers me the ternary operator. I have used the ternary, so I'm not sure what you'd like here. Should I redo it as if statements or should I just review how I did the ternary?